src/components/database/DatabaseOverview.vue

```diff
@@ -0,0 +1,250 @@
+<template>
+    <div>
+        <v-unipept-card class="mb-6">
+            <v-card-title class="text-h5 font-weight-bold">
+                <v-icon icon="mdi-database" size="30" class="me-2" color="primary"/>
+                Custom Databases
+            </v-card-title>
+            <v-card-text class="text-body-1">
+                Below you can find a list of all custom databases that are currently registered to this application.
+                To create a new custom database, press the floating button at the bottom. A wizard will guide you
```

Comment:
There is no floating action button anymore in this overview ;) The description should be updated.